javascript/src/main/components/OfficeHours/OfficeHourTable.js

```diff
@@ -44,6 +51,8 @@ export default ({officeHours,admin,deleteOfficeHour}) => {
         });
     }
 
+    //call formatter function
```

Comment:
Small nitpick but this comment seems to be unnecessary.

Reply:
Thanks for addressing @jvoucsb 's comment